dbus.c

```diff
@@ -55,6 +60,7 @@ const zend_function_entry dbus_funcs_dbus[] = {
 	PHP_ME(Dbus, requestName, arginfo_void, ZEND_ACC_PUBLIC)
 	PHP_ME(Dbus, registerObject, arginfo_void, ZEND_ACC_PUBLIC)
 	PHP_ME(Dbus, createProxy, arginfo_void, ZEND_ACC_PUBLIC)
+	PHP_ME(Dbus, setDefaultTimeout, arginfo_dbus_setDefaultTimeout, ZEND_ACC_PUBLIC)
```

Comment:
I'm of the opinion that methods that set a time out, should include the unit. You've added `ms` to the flag that controls this, so I would also recommend you change the method to `setDefaultTimeoutMS`.